code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -205,6 +205,13 @@ GLOBAL_LIST_INIT(wood_recipes, list ( \
 	new/datum/stack_recipe("wooden crate", /obj/structure/closet/crate/wooden, 6, time = 50, one_per_turf = TRUE, on_floor = TRUE),\
 	new/datum/stack_recipe("baseball bat", /obj/item/melee/baseball_bat, 5, time = 15),\
 	new/datum/stack_recipe("loom", /obj/structure/loom, 10, time = 15, one_per_turf = TRUE, on_floor = TRUE), \
+	null, \
```

Comment:
what's with the null entries?

Reply:
They add lines in the build menu to group items in the build menu. I copied the syntax from the sofa build options higher up in the file. I can remove them if you want.

Reply:
no that's fine, didn't realise they were used for formatting